frontend/src/pages/SingleCardPage.js

```diff
@@ -0,0 +1,15 @@
+import { useParams } from 'react-router-dom'
+
+export default function SingleCardPage(){
+
+    const { name, id } = useParams();
+
+
+    return(
+        <div>
+        <h2>{name}</h2>
+        <h3>{id}</h3>
+        </div>
```

Comment:
```suggestion
        <section>
        <p>{name}</p>
        <p>{id}</p>
        </section>
```

Reply:
thanks, I will consider this for the next pr